packages/fluentui/docs/src/examples/components/Card/Usage/index.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+
+import ComponentExample from '../../../../components/ComponentDoc/ComponentExample';
+import ExampleSection from '../../../../components/ComponentDoc/ExampleSection';
+
+const Usage = () => (
+  <ExampleSection title="Usage">
+    <ComponentExample
+      title="Header"
+      description="Simple card with header only"
+      examplePath="components/Card/Usage/CardExampleSimpleOnlyHeader"
```

Comment:
```suggestion
      description="A card can contain a header slot."
      examplePath="components/Card/Usage/CardExampleHeader"
```

Reply:
Applicable to other examples. Example that features only slots usage should be moved to `Slots` section